api/appStore/InstalledAppRestHandler.go

```diff
@@ -396,6 +401,17 @@ func (handler *InstalledAppRestHandlerImpl) DeployBulk(w http.ResponseWriter, r
 	common.WriteJsonResp(w, err, res, http.StatusOK)
 }
 
+func (handler *InstalledAppRestHandlerImpl) checkForHelmDeployAuth(rbacObject string, request chartGroup.ChartGroupInstallRequest, token string) bool {
+	rbacObject, rbacObject2 := handler.enforcerUtil.GetHelmObjectByProjectIdAndEnvId(request.ProjectId, request.ChartGroupInstallChartRequest[0].EnvironmentId)
```

Comment:
why are we checking for only first environmentId and not for others. 
1) We will have to apply rbac rbac for each environmentId.
2) use EnforceInBatch for this
3) handle case if user has access to only some environments, sync with product for same